2024-02-04 15:15:30.247 [info] Extension host with pid 7235 started
2024-02-04 15:15:30.247 [info] Skipping acquiring lock for /home/elisena/.config/Code/User/workspaceStorage/6442a3781aac7f0018a3b769379cf1e2.
2024-02-04 15:15:30.417 [info] ExtensionService#_doActivateExtension ms-python.python, startup: false, activationEvent: 'onLanguage:python'
2024-02-04 15:15:30.472 [info] ExtensionService#_doActivateExtension ms-toolsai.jupyter, startup: false, activationEvent: 'onLanguage:python'
2024-02-04 15:15:30.475 [info] ExtensionService#_doActivateExtension vscode.emmet, startup: false, activationEvent: 'onLanguage'
2024-02-04 15:15:31.020 [info] ExtensionService#_doActivateExtension ms-toolsai.jupyter-renderers, startup: false, activationEvent: 'api', root cause: ms-toolsai.jupyter
2024-02-04 15:15:31.073 [info] ExtensionService#_doActivateExtension vscode.git-base, startup: true, activationEvent: '*', root cause: vscode.git
2024-02-04 15:15:31.075 [info] ExtensionService#_doActivateExtension mhutchie.git-graph, startup: true, activationEvent: '*'
2024-02-04 15:15:31.183 [info] ExtensionService#_doActivateExtension vscode.git, startup: true, activationEvent: '*'
2024-02-04 15:15:31.207 [info] ExtensionService#_doActivateExtension vscode.github, startup: true, activationEvent: '*'
2024-02-04 15:15:31.284 [info] Eager extensions activated
2024-02-04 15:15:31.356 [info] ExtensionService#_doActivateExtension vscode.debug-auto-launch, startup: false, activationEvent: 'onStartupFinished'
2024-02-04 15:15:31.358 [info] ExtensionService#_doActivateExtension vscode.merge-conflict, startup: false, activationEvent: 'onStartupFinished'
2024-02-04 15:15:31.364 [info] ExtensionService#_doActivateExtension Catppuccin.catppuccin-vsc, startup: false, activationEvent: 'onStartupFinished'
2024-02-04 15:15:31.368 [info] ExtensionService#_doActivateExtension eamodio.gitlens, startup: false, activationEvent: 'onStartupFinished'
2024-02-04 15:15:31.423 [info] ExtensionService#_doActivateExtension esbenp.prettier-vscode, startup: false, activationEvent: 'onStartupFinished'
2024-02-04 15:15:31.467 [info] ExtensionService#_doActivateExtension pdconsec.vscode-print, startup: false, activationEvent: 'onStartupFinished'
2024-02-04 15:15:31.506 [info] ExtensionService#_doActivateExtension phind.phind, startup: false, activationEvent: 'onStartupFinished'
2024-02-04 15:15:32.032 [info] ExtensionService#_doActivateExtension streetsidesoftware.code-spell-checker, startup: false, activationEvent: 'onStartupFinished'
2024-02-04 15:15:32.072 [info] ExtensionService#_doActivateExtension vscodevim.vim, startup: false, activationEvent: 'onStartupFinished'
2024-02-04 15:15:32.515 [error] [ms-python.python] provider FAILED
2024-02-04 15:15:32.515 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:15:32.541 [error] [ms-python.python] provider FAILED
2024-02-04 15:15:32.541 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:15:32.562 [error] [ms-python.python] provider FAILED
2024-02-04 15:15:32.562 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:15:32.564 [info] ExtensionService#_doActivateExtension vscode.github-authentication, startup: false, activationEvent: 'onAuthenticationRequest:github'
2024-02-04 15:15:32.768 [error] [ms-python.python] provider FAILED
2024-02-04 15:15:32.768 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:15:32.768 [error] [ms-python.python] provider FAILED
2024-02-04 15:15:32.768 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:15:33.230 [info] ExtensionService#_doActivateExtension vscode.json-language-features, startup: false, activationEvent: 'onLanguage:jsonc'
2024-02-04 15:15:33.244 [info] ExtensionService#_doActivateExtension vscode.typescript-language-features, startup: false, activationEvent: 'onLanguage:jsonc'
2024-02-04 15:15:33.315 [error] [ms-python.python] provider FAILED
2024-02-04 15:15:33.315 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:15:33.316 [error] [ms-python.python] provider FAILED
2024-02-04 15:15:33.316 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:15:33.339 [error] [ms-python.python] provider FAILED
2024-02-04 15:15:33.339 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:15:33.403 [error] [ms-python.python] provider FAILED
2024-02-04 15:15:33.403 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:15:33.593 [error] [ms-python.python] provider FAILED
2024-02-04 15:15:33.593 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:15:33.595 [error] [ms-python.python] provider FAILED
2024-02-04 15:15:33.595 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:15:37.604 [error] [ms-python.python] provider FAILED
2024-02-04 15:15:37.604 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:15:37.854 [error] [ms-python.python] provider FAILED
2024-02-04 15:15:37.855 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:04.898 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:04.898 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:05.102 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:05.102 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:05.355 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:05.355 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:07.764 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:07.764 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:07.770 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:07.770 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:08.051 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:08.051 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:10.508 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:10.508 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:10.969 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:10.969 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:13.459 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:13.459 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:13.617 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:13.617 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:13.869 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:13.869 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:16.533 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:16.533 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:16.968 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:16.968 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:17.940 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:17.940 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:21.086 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:21.086 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:21.440 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:21.440 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:21.691 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:21.691 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:21.997 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:21.997 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:22.210 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:22.210 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:22.380 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:22.380 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:22.529 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:22.529 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:22.781 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:22.781 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:24.714 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:24.714 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:24.719 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:24.719 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:25.078 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:25.078 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:30.691 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:30.691 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:30.907 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:30.907 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:31.457 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:31.457 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:31.857 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:31.857 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:32.123 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:32.123 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:37.884 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:37.884 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:46.156 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:46.156 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:46.856 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:46.856 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:46.886 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:46.886 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:47.131 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:47.131 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:49.090 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:49.090 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:49.341 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:49.348 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:49.548 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:49.548 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:49.799 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:49.799 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:54.606 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:54.606 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:55.421 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:55.421 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:56.283 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:56.283 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:56.296 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:56.296 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:56.541 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:56.541 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:57.771 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:57.771 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:58.021 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:58.021 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:58.181 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:58.181 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:58.434 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:58.434 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:58.562 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:58.562 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:58.814 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:58.814 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:58.886 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:58.886 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:59.081 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:59.081 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:59.272 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:59.272 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:59.471 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:59.471 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:59.670 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:59.670 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:59.895 [error] [ms-python.python] provider FAILED
2024-02-04 15:16:59.895 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:17:00.095 [error] [ms-python.python] provider FAILED
2024-02-04 15:17:00.095 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:17:00.348 [error] [ms-python.python] provider FAILED
2024-02-04 15:17:00.348 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:17:01.939 [error] [ms-python.python] provider FAILED
2024-02-04 15:17:01.939 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:17:02.022 [error] [ms-python.python] provider FAILED
2024-02-04 15:17:02.022 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:17:02.270 [error] [ms-python.python] provider FAILED
2024-02-04 15:17:02.270 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:17:02.589 [error] [ms-python.python] provider FAILED
2024-02-04 15:17:02.589 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:17:03.339 [error] [ms-python.python] provider FAILED
2024-02-04 15:17:03.346 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:17:03.590 [error] [ms-python.python] provider FAILED
2024-02-04 15:17:03.590 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:17:03.716 [error] [ms-python.python] provider FAILED
2024-02-04 15:17:03.716 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:17:03.921 [error] [ms-python.python] provider FAILED
2024-02-04 15:17:03.921 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:17:04.173 [error] [ms-python.python] provider FAILED
2024-02-04 15:17:04.173 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:17:07.867 [error] [ms-python.python] provider FAILED
2024-02-04 15:17:07.867 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:17:07.873 [error] [ms-python.python] provider FAILED
2024-02-04 15:17:07.873 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:17:08.127 [error] [ms-python.python] provider FAILED
2024-02-04 15:17:08.127 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:17:08.329 [error] [ms-python.python] provider FAILED
2024-02-04 15:17:08.329 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:17:08.762 [error] [ms-python.python] provider FAILED
2024-02-04 15:17:08.762 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:17:10.845 [error] [ms-python.python] provider FAILED
2024-02-04 15:17:10.845 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:17:10.860 [error] [ms-python.python] provider FAILED
2024-02-04 15:17:10.860 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:17:18.668 [warning] TextEditor is closed/disposed
2024-02-04 15:17:18.755 [error] [ms-python.python] provider FAILED
2024-02-04 15:17:18.755 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:17:18.756 [error] [ms-python.python] provider FAILED
2024-02-04 15:17:18.756 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:17:20.184 [warning] TextEditor is closed/disposed
2024-02-04 15:17:20.263 [error] [ms-python.python] provider FAILED
2024-02-04 15:17:20.263 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:17:20.905 [error] [ms-python.python] provider FAILED
2024-02-04 15:17:20.905 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:19:15.565 [error] [ms-python.python] provider FAILED
2024-02-04 15:19:15.565 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:19:15.566 [error] [ms-python.python] provider FAILED
2024-02-04 15:19:15.566 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:19:33.571 [warning] TextEditor is closed/disposed
2024-02-04 15:19:33.625 [error] [ms-python.python] provider FAILED
2024-02-04 15:19:33.625 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:19:33.626 [error] [ms-python.python] provider FAILED
2024-02-04 15:19:33.626 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:19:34.107 [error] [ms-python.python] provider FAILED
2024-02-04 15:19:34.108 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:19:40.390 [error] [ms-python.python] provider FAILED
2024-02-04 15:19:40.390 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:19:40.390 [error] [ms-python.python] provider FAILED
2024-02-04 15:19:40.391 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:19:40.847 [error] [ms-python.python] provider FAILED
2024-02-04 15:19:40.847 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:19:41.119 [error] [ms-python.python] provider FAILED
2024-02-04 15:19:41.119 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:19:42.603 [error] [ms-python.python] provider FAILED
2024-02-04 15:19:42.603 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:19:43.966 [error] [ms-python.python] provider FAILED
2024-02-04 15:19:43.966 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:19:49.383 [error] [ms-python.python] provider FAILED
2024-02-04 15:19:49.383 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:19:49.384 [error] [ms-python.python] provider FAILED
2024-02-04 15:19:49.384 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:19:49.845 [error] [ms-python.python] provider FAILED
2024-02-04 15:19:49.845 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:19:53.077 [error] [ms-python.python] provider FAILED
2024-02-04 15:19:53.077 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:19:53.329 [error] [ms-python.python] provider FAILED
2024-02-04 15:19:53.329 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:19:54.582 [warning] TextEditor is closed/disposed
2024-02-04 15:19:54.597 [error] [ms-python.python] provider FAILED
2024-02-04 15:19:54.597 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:20:06.361 [error] [ms-python.python] provider FAILED
2024-02-04 15:20:06.361 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:20:37.228 [warning] TextEditor is closed/disposed
2024-02-04 15:20:37.248 [error] [ms-python.python] provider FAILED
2024-02-04 15:20:37.249 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:20:37.496 [error] [ms-python.python] provider FAILED
2024-02-04 15:20:37.496 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:20:38.465 [warning] TextEditor is closed/disposed
2024-02-04 15:20:38.480 [error] [ms-python.python] provider FAILED
2024-02-04 15:20:38.480 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:21:22.296 [warning] TextEditor is closed/disposed
2024-02-04 15:21:22.340 [error] [ms-python.python] provider FAILED
2024-02-04 15:21:22.341 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:21:28.607 [error] [ms-python.python] provider FAILED
2024-02-04 15:21:28.607 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:21:28.608 [error] [ms-python.python] provider FAILED
2024-02-04 15:21:28.608 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:21:29.637 [warning] TextEditor is closed/disposed
2024-02-04 15:21:29.756 [info] ExtensionService#_doActivateExtension vscode.extension-editing, startup: false, activationEvent: 'onLanguage:markdown'
2024-02-04 15:21:29.784 [info] ExtensionService#_doActivateExtension vscode.markdown-language-features, startup: false, activationEvent: 'onLanguage:markdown'
2024-02-04 15:21:30.639 [info] ExtensionService#_doActivateExtension vscode.markdown-math, startup: false, activationEvent: 'api', root cause: vscode.markdown-language-features
2024-02-04 15:21:30.691 [warning] TextEditor is closed/disposed
2024-02-04 15:21:43.069 [error] [ms-python.python] provider FAILED
2024-02-04 15:21:43.069 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:21:43.071 [error] [ms-python.python] provider FAILED
2024-02-04 15:21:43.071 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:21:44.316 [error] [ms-python.python] provider FAILED
2024-02-04 15:21:44.316 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:23:05.715 [error] [ms-python.python] provider FAILED
2024-02-04 15:23:05.716 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:23:07.792 [error] [ms-python.python] provider FAILED
2024-02-04 15:23:07.792 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:23:08.670 [error] [ms-python.python] provider FAILED
2024-02-04 15:23:08.670 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:23:33.178 [error] [ms-python.python] provider FAILED
2024-02-04 15:23:33.178 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:23:33.267 [error] [ms-python.python] provider FAILED
2024-02-04 15:23:33.267 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:23:33.428 [error] [ms-python.python] provider FAILED
2024-02-04 15:23:33.428 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:23:42.920 [warning] TextEditor is closed/disposed
2024-02-04 15:23:43.023 [error] [ms-python.python] provider FAILED
2024-02-04 15:23:43.024 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:23:43.025 [error] [ms-python.python] provider FAILED
2024-02-04 15:23:43.025 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:23:43.613 [error] [ms-python.python] provider FAILED
2024-02-04 15:23:43.614 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:23:51.972 [error] [ms-python.python] provider FAILED
2024-02-04 15:23:51.972 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:24:08.288 [error] [ms-python.python] provider FAILED
2024-02-04 15:24:08.288 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:24:08.709 [error] [ms-python.python] provider FAILED
2024-02-04 15:24:08.709 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:24:09.531 [error] [ms-python.python] provider FAILED
2024-02-04 15:24:09.531 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:24:47.785 [warning] TextEditor is closed/disposed
2024-02-04 15:24:47.827 [error] [ms-python.python] provider FAILED
2024-02-04 15:24:47.827 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:24:47.902 [warning] Illegal argument: TextEditor(vs.editor.ICodeEditor:2,$model22)
2024-02-04 15:24:47.902 [warning] Illegal argument: TextEditor(vs.editor.ICodeEditor:2,$model22)
2024-02-04 15:24:47.902 [warning] Illegal argument: TextEditor(vs.editor.ICodeEditor:2,$model22)
2024-02-04 15:24:47.902 [warning] Illegal argument: TextEditor(vs.editor.ICodeEditor:2,$model22)
2024-02-04 15:24:47.902 [warning] Illegal argument: TextEditor(vs.editor.ICodeEditor:2,$model22)
2024-02-04 18:41:57.912 [error] [ms-python.python] provider FAILED
2024-02-04 18:41:57.912 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 18:53:07.316 [error] [ms-python.python] provider FAILED
2024-02-04 18:53:07.317 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 18:54:04.023 [error] [ms-python.python] provider FAILED
2024-02-04 18:54:04.023 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 18:55:40.443 [error] [ms-python.python] provider FAILED
2024-02-04 18:55:40.444 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
